package/MDAnalysis/coordinates/XTC.py

```diff
@@ -26,15 +37,15 @@ class XTCWriter(XDRBaseWriter):
     they require significantly less disk space and the loss of precision is
     usually not a problem.
 
-    Parameter
-    ---------
-    filename : str
+    Parameters
+    ----------
+    filename: str
```

Comment:
I'm not a huge fan of it either but I'm just going with what other use to make it easier for other who want to contribute.